book/propel.rst

```diff
@@ -60,7 +45,13 @@ configuration file (``config.yml``):
             password: "%database_password%"
             dsn:      "%database_driver%:host=%database_host%;dbname=%database_name%;charset=%database_charset%"
 
-Now that Propel knows about your database, Symfony can create the database for
+.. note::
+
+    Defining the configuration via ``parameters.yml`` is a
+    :ref:`Symfony Framework Best Practice <best-practices-canonical-parameters>`,
+    feel free to do it differently if that suits your application better.
+
+Now that Propel knows about your database, it can create the database for
```

Comment:
Propel isn't a part of the docs anymore, shouldn't we think about dropping this?